settingengine.go

```diff
@@ -239,3 +241,14 @@ func (e *SettingEngine) DisableSRTPReplayProtection(isDisabled bool) {
 func (e *SettingEngine) DisableSRTCPReplayProtection(isDisabled bool) {
 	e.disableSRTCPReplayProtection = isDisabled
 }
+
+// AddSDPExtensions adds available and offered extensions for media type.
```

Comment:
:+1: LGTM!

Reply:
I would make sure the comment mentions
* Don't include the ID, we generate those for you
* If answering not all may be included